test/uk/gov/hmrc/transitmovementsauditing/controllers/AuditControllerSpec.scala

```diff
@@ -41,11 +41,9 @@ class AuditControllerSpec extends AnyFreeSpec with Matchers with TestActorSystem
   private val controller = new AuditController(controllerComponentWithTempFile)(materializer)
 
   "POST /" - {
-
     "return 202" in {
       val result = controller.post(AmendmentAcceptance)(fakeRequest)
```

Comment:
How can we test that the string is "AmendmentAcceptance" is being converted to AuditType.AmendmentAcceptance

Reply:
I think the only was is via an integration test, unless anyone knows of a better approach?

Reply:
Can we directly unit test the binder?

Reply:
Yes, thats a better idea, will add one for the binder